### PR TITLE
Add MenuMeters

### PR DESCRIPTION
**Pull request by dufferzafar**
This version is for El Capitan. 

For older versions, see: http://www.ragingmenace.com/software/menumeters/

### CODE REVIEW

**Comment by dufferzafar** [comment]
iStatsMenu (already on this list) is a similar application, but it installs a helper which requires root permissions (which not everyone may have.)

**Comment by dkhamsing** [comment]
@dufferzafar looks pretty good
can you rebase/fix the conflicts?

**Comment by dufferzafar** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -252,11 +252,13 @@
 - [LaunchControl](http://www.soma-zone.com/LaunchControl/) - Create, manage and debug launchd services. ![Freeware][Freeware Icon]
 - [Loading](http://bonzaiapps.com) - See when apps are using your network in your Mac menubar. [![Open-Source Software][OSS Icon]](https://github.com/BonzaiThePenguin/Loading/) ![Freeware][Freeware Icon]
 - [Little Snitch](https://www.obdev.at/products/littlesnitch/index.html) - Protect your privacy.
+<<<<<<< ec51ef1a7f25686c4e2a08b4476076575c669358
```
Comment (iCHAIT) [inline README.md]: You need to remove this.

File: README.md [hunk 2]
```diff
@@ -252,11 +252,13 @@
...
 - [Mackup](https://github.com/lra/mackup) - Keep your application settings in sync. [![Open-Source Software][OSS Icon]](https://github.com/lra/mackup) ![Freeware][Freeware Icon]
 - [MacPass](http://mstarke.github.io/MacPass/) - Password Manager. [![Open-Source Software][OSS Icon]](https://github.com/mstarke/MacPass) ![Freeware][Freeware Icon]
 - [Media Converter](http://media-converter.sourceforge.net/) - Simple (drag and drop) but advanced media conversion. [![Open-Source Software][OSS Icon]](https://sourceforge.net/p/media-converter/code/HEAD/tree/) ![Freeware][Freeware Icon]
 - [Menubar Colors](https://github.com/nvzqz/Menubar-Colors) - Convenient access to the system color panel. [![Open-Source Software][OSS Icon]](https://github.com/nvzqz/Menubar-Colors) ![Freeware][Freeware Icon]
+- [MenuMeters](http://member.ipmu.jp/yuji.tachikawa/MenuMetersElCapitan/) - A set of CPU, memory, disk, and network monitoring tools for Mac OS X. [![Open-Source Software][OSS Icon]](https://github.com/yujitach/MenuMeters)
 - [Monodraw](http://monodraw.helftone.com/) - A powerful ASCII art editor.
```
Comment (iCHAIT) [inline README.md]: Replace `Mac OS X` with `macOS`

**Comment by dufferzafar** [comment]
@iCHAIT Done!

**Comment by dkhamsing** [comment]
Thx

**Pull request merged**